src/Serilog.Sinks.MSSqlServer/Sinks/MSSqlServer/Platform/SqlInsertBatchWriter.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Serilog.Debugging;
+using Serilog.Events;
+using Serilog.Sinks.MSSqlServer.Output;
+using static System.FormattableString;
+
+namespace Serilog.Sinks.MSSqlServer.Platform
+{
+    internal class SqlInsertBatchWriter : ISqlBulkBatchWriter
```

Comment:
Could we rewrite `SqlLogEventWriter` to use this class so we don't need to maintain 2 classes with nearly identical code?

@ckadluba Do you think that would be a good idea?

Reply:
Merged the `SqlLogEventWriter` and `SqlInsertBatchWriter` into one `SqlInsertStatementWriter` ([4db4513](https://github.com/serilog-mssql/serilog-sinks-mssqlserver/pull/510/commits/4db45135e57ad4b72b196a43e3d1bfa9bd78970d)).  Let me know if you like the change if not I can always revert it.  Feel free to provide additional direction here as well.